src/components/Table/TableBase.tsx

```diff
@@ -186,10 +193,12 @@ export class TableBase<T extends {}> extends React.Component<
 			this.setRowSelection(checkedItems);
```

Comment:
I think we also need to doublecheck how selection works.
* We need to update `toggleChecked`, since it does include a client side sorting and assumes it has all the data while trying to map the checkbox the user clicked and the item that the row was about.
* We probably also need to check `toggleAllChecked` as well, and maybe in server side more it should be returning `all`, to let the consumer know that they need to query and fetch AAAAAALLLLLLLLLL item IDs from the backend.

Reply:
Work in progress. I need to merge this PR first to be able to do something like this in the following PR:

isDisabled & actionFn:
   * isServerSide: false =>  T[]
   * isServerSide: true => 
   * selecting N items: { id: { $in: [] } }
   * select-all: { $and: [{ belongs_to__application: xyz }, searchFilters] }